index.js

```diff
@@ -0,0 +1,31 @@
+const langSec = document.querySelector(".language-section");
+const buttons = document.querySelectorAll(".button");
+const textTitle = document.querySelector(".title");
+const textDesc = document.querySelector(".description");
+
+buttons.forEach(button => {
+    button.addEventListener('click', () => {
+        langSec.querySelector(".active").classList.remove(".active");
+        button.classList.add("active");
+
+        const attr = button.getAttribute("data-language");
+
+        textTitle.textContent = data[attr].title;
+        textDesc.textContent = data[attr].description;
+    });
+});
+
+const data = {
+    "english": {
+        "title": "Translating static text.",
+        "description": "Translating static text"
+    },
+    "french": {
+        "title": "Traduire du texte statique.",
+        "description": "Traduire du texte statique."
+    },
+    "german": {
+        "title": "Statischen Text übersetzen.",
+        "description": "Statischen Text übersetzen."
+    }
+}
```

Comment:
```suggestion
}

```

Missing a return, which means either you did not come back to this file and hit save after adding the editor config file. Or you did not install the editor config plugin in your editor.

Reply:
I have added the editor config file but I'm not for sure if I saved after adding that file. I do have the editor config plugin installed.

I tried to add an empty return which seems like it doesn't show up unless when in the editor.